plat/arm/board/fvp/aarch64/fvp_helpers.S

```diff
@@ -183,14 +183,12 @@ endfunc plat_is_my_cpu_primary
 	 *
 	 * Function to calculate the core position on FVP.
 	 *
-	 * (ClusterId * FVP_MAX_CPUS_PER_CLUSTER) +
-	 * (CPUId * FVP_MAX_PE_PER_CPU) +
+	 * (ClusterId * FVP_MAX_CPUS_PER_CLUSTER + CPUId) *
```

Comment:
I'd prefer to see the actual formula listed here, followed by the simplified one:

```
(ClusterId * FVP_MAX_CPU_PER_CLUSTER * FVP_MAX_PE_PER_CPU) +
(CPUId * FVP_MAX_PE_PER_CPU) +
ThreadId

which can be simplified as:

(((ClusterId * FVP_MAX_CPU_PER_CLUSTER) + CPUId) * FVP_MAX_PE_PER_CPU) +
TreadId
```

Reply:
I have added these in annotation. I have to delete a pair of parentheses because of the limit of 80 characters per line.